datasets/ru/rupep/crawler.py

```diff
@@ -53,14 +97,195 @@ def parse_date(date):
     return h.parse_date(date, FORMATS)
 
 
-def crawl_person(context: Context, data: Dict[str, Any]):
+def crawl_person_person_relation(context: Context, entity: Entity, rel_data: dict):
+    other_pep = rel_data.pop("is_pep", False)
+    other_wdid = clean_wdid(rel_data.pop("person_wikidata_id"))
+    other = context.make("Person")
+    other.id = person_id(context, rel_data.pop("person_id"), other_wdid)
+    if other.id is None:
+        return
+    other.add("name", rel_data.pop("person_en", None), lang="eng")
+    other.add("name", rel_data.pop("person_ru", None), lang="rus")
+    other.add("wikidataId", other_wdid)
+
+    rel_type = rel_data.pop("relationship_type_en", None)
+    rel_type_ru = rel_data.pop("relationship_type_ru", None)
+    rel_type = rel_type or rel_type_ru
+    res = context.lookup("person_person_relations", rel_type)
+    if res is None:
+        context.log.warn(
+            "Unknown person/person relation type",
+            rel_type=rel_type,
+            entity=entity,
+            other=other,
+        )
+        return
+
+    # print("LINK", (entity.id, other.id))
+    id_a, id_b = sorted((entity.id, other.id))
+    rel = context.make(res.schema)
+    id_a_short = short_id(context, id_a)
+    id_b_short = short_id(context, id_b)
+    rel.id = context.make_slug(id_a_short, res.schema, id_b_short)
+    rel.add(res.from_prop, id_a)
+    rel.add(res.to_prop, id_b)
+    rel.add(res.desc_prop, rel_type)
+    rel.add("modifiedAt", parse_date(rel_data.pop("date_confirmed")))
+    rel.add("startDate", parse_date(rel_data.pop("date_established")))
+    rel.add("endDate", parse_date(rel_data.pop("date_finished")))
+
+    context.audit_data(rel_data)
+    context.emit(other, target=other_pep)
+    context.emit(rel)
+
+
+def crawl_company_person_relation(
+    context: Context, company: Entity, person: Entity, rel_data: dict
+):
```

Comment:
check if all rel_date fields from the person match rel_data fields from the company